<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="EONPC-18" name="com.lambdatest.SignUpTest" tests="14" failures="2" timestamp="2019-10-09T11:07:47 IST" time="200.951" errors="0">
  <testcase name="inValidPasswordTest" time="14.855" classname="com.lambdatest.SignUpTest"/>
  <testcase name="emptyPhoneTest" time="15.030" classname="com.lambdatest.SignUpTest"/>
  <testcase name="emptyEmailTest" time="14.652" classname="com.lambdatest.SignUpTest"/>
  <testcase name="inValidPhoneTest" time="14.116" classname="com.lambdatest.SignUpTest"/>
  <testcase name="privacyPolicyRedirectionTest" time="9.765" classname="com.lambdatest.SignUpTest"/>
  <testcase name="verifyElemntsOnPageTest" time="9.150" classname="com.lambdatest.SignUpTest"/>
  <testcase name="emptyPasswordTest" time="15.004" classname="com.lambdatest.SignUpTest"/>
  <testcase name="uncheckedTerms" time="12.445" classname="com.lambdatest.SignUpTest"/>
  <testcase name="termsRedirectionTest" time="10.092" classname="com.lambdatest.SignUpTest"/>
  <testcase name="validRegistrationTest" time="29.334" classname="com.lambdatest.SignUpTest">
    <failure type="java.lang.AssertionError" message="expected [Login - LambdaTest] but found [Verify Your Email Address - LambdaTest]">
      <![CDATA[java.lang.AssertionError: expected [Login - LambdaTest] but found [Verify Your Email Address - LambdaTest]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.lambdatest.SignUpTest.validRegistrationTest(SignUpTest.java:183)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validRegistrationTest -->
  <testcase name="emptyNameTest" time="14.820" classname="com.lambdatest.SignUpTest"/>
  <testcase name="emptyCompanyNameTest" time="19.720" classname="com.lambdatest.SignUpTest">
    <failure type="java.lang.AssertionError" message="expected [Login - LambdaTest] but found [Verify Your Email Address - LambdaTest]">
      <![CDATA[java.lang.AssertionError: expected [Login - LambdaTest] but found [Verify Your Email Address - LambdaTest]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.lambdatest.SignUpTest.emptyCompanyNameTest(SignUpTest.java:225)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- emptyCompanyNameTest -->
  <testcase name="loginRedirectionTest" time="7.899" classname="com.lambdatest.SignUpTest"/>
  <testcase name="invalidEmailTest" time="14.069" classname="com.lambdatest.SignUpTest"/>
</testsuite> <!-- com.lambdatest.SignUpTest -->
